workers/unity/Assets/Fps/Scripts/UI/Managers/DeploymentListScreenManager.cs

```diff
@@ -51,18 +50,27 @@ public string GetChosenDeployment()
 
         public void SetDeployments(DeploymentData[] deployments)
         {
+            var highlightedDeploymentName = string.Empty;
+            if (DeploymentList != null && currentlyHighlightedEntry != -1)
+            {
+                highlightedDeploymentName = DeploymentList[currentlyHighlightedEntry].Name;
+            }
+
+            currentlyHighlightedEntry = -1;
```

Comment:
Make make this a nullable instead of a magic -1 for no value?

Reply:
discussed offline, will make the logic a bit clearer but not make it nullable